packages/dropzone/stories/dropzone.stories.ts

```diff
@@ -44,48 +50,140 @@ type StoryArgs = {
-                        <sp-link>Select a File</sp-link>
-                        from your computer
-                    </label>
-                    <input type="file" id="file-input" style="display: none" />
-                </div>
-                <div slot="description">
-                    or
-                    <sp-link href="http://stock.adobe.com" target="blank">
-                        Search Adobe Stock
-                    </sp-link>
-                </div>
             </sp-illustrated-message>
+            <div>
+                <label for="file-input">
+                    <sp-link
+                        href="javascript:;"
+                        onclick="this.parentElement.nextElementSibling.click()"
+                    >
+                        Select a File
+                    </sp-link>
+                    from your computer
+                </label>
+                <input type="file" id="file-input" style="display: none" />
+            </div>
+            <div>
+                or
+                <sp-link href="http://stock.adobe.com" target="blank">
+                    Search Adobe Stock
+                </sp-link>
+            </div>
         </sp-dropzone>
     `;
 };
 
 export const Dragged = (args: StoryArgs): TemplateResult => {
     return html`
         <sp-dropzone id="dropzone" tabindex="0" ?dragged=${args.isDragged}>
-            <sp-illustrated-message heading="Drag and Drop Your File" cta>
+            <sp-illustrated-message heading="Drag and Drop Your File">
                 ${illustration}
-                <div slot="description">
-                    <label for="file-input">
-                        <sp-link>Select a File</sp-link>
-                        from your computer
-                    </label>
-                    <input type="file" id="file-input" style="display: none" />
-                </div>
-                <div slot="description">
-                    or
-                    <sp-link href="http://stock.adobe.com" target="blank">
-                        Search Adobe Stock
-                    </sp-link>
-                </div>
             </sp-illustrated-message>
+            <div>
+                <label for="file-input">
+                    <sp-link
+                        href="javascript:;"
+                        onclick="this.parentElement.nextElementSibling.click()"
+                    >
+                        Select a File
+                    </sp-link>
+                    from your computer
+                </label>
+                <input type="file" id="file-input" style="display: none" />
+            </div>
+            <div>
+                or
+                <sp-link href="http://stock.adobe.com" target="blank">
+                    Search Adobe Stock
+                </sp-link>
+            </div>
         </sp-dropzone>
     `;
 };
 Dragged.args = {
     isDragged: true,
 };
+
+class ControlledDropzone extends LitElement {
```

Comment:
See PR description for more examples w/ screen reader

Screenshot:

![dropzone_controlled_example](https://github.com/adobe/spectrum-web-components/assets/24359834/1ed9b684-0e17-4a01-92ae-38b80613ce87)